src/main/webapp/app/exam/manage/exams/exam-update.component.ts

```diff
@@ -45,13 +55,22 @@ export class ExamUpdateComponent implements OnInit {
     ngOnInit(): void {
         this.route.data.subscribe(({ exam }) => {
             this.exam = exam;
+
+            // Tap the URL to determine, if the Exam should be imported
+            this.route.url.pipe(tap((segments) => (this.isImport = segments.some((segment) => segment.path === 'import')))).subscribe();
```

Comment:
`tap` is not invoked, if you don't subscribe, right?

Reply:
yes, afaik.